doc/api/crypto.md

```diff
@@ -2150,11 +2150,11 @@ added: v10.5.0
 - `keylen` {number}
 - `options` {Object}
   - `N` {number} CPU/memory cost parameter. Must be a power of two greater
-                 than one. **Default:** `16384`.
+    than one. **Default:** `16384`.
   - `r` {number} Block size parameter. **Default:** `8`.
   - `p` {number} Parallelization parameter. **Default:** `1`.
   - `maxmem` {number} Memory upper bound. It is an error when (approximately)
-                      `128*N*r > maxmem` **Default:** `32 * 1024 * 1024`.
+    `128 * N * r > maxmem` **Default:** `32 * 1024 * 1024`.
```

Comment:
Sorry for not spotting this first time: it seems we lack the period between `` maxmem` `` and `**Default:**`. Can you add it since we are already here?)

Reply:
"it seems we lack the **period** between `maxmem` and `**Default:**`" What do you mean on "period" word? Could you please write an example?

Reply:
Sorry for the ambiguity) I mean we need this fix:

`` maxmem`. **Default:** ``

Reply:
I see. Sure.